packages/fx-core/src/component/generator/officeXMLAddin/projectConfig.ts

```diff
@@ -1,208 +1,189 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
OK, I see this config file store the information for both Office Addin and Office XML Addin.

Maybe we can consider move this file's location from folder `officeXMLAddin` to `officeAddin`?\

Since for long long long term consideration, office xml addin will be not-recommanded.

Reply:
Yeah, we can improve code in the future.